### DISCUSSION TITLE
free Herokuapp going away

### DISCUSSION THREAD

**Discussion by SliceOfBread** [bafolts/terraforming-mars#4822] [General]
You've probably seen the announcement that the free hobby Heroku is going away Nov 28, 2022. https://blog.heroku.com/next-chapter
I'm sure at least a few people are using this to run their own server (I was for a time).
The Wiki and a few documents will (eventually) need to remove reference to this service. Are there any replacements services (other than the $16/mo heroku)?

**Comment by kberg** [comment]
None yet, but I'm sure this is being discussed on the other discords, too. I do wonder what the docker options look like, in terms of platforms and costs. I don't think we have very detailed instructions for other services like we do Heroku.

I'll keep the hobby instance I have for now.

**Comment by Gugatec** [comment]
There is a couple of ways of doing it. The one I'm currently using is Oracle OCI free tier running docker and connecting via cloudflare (requires a domain name), but you can install a DDNS and use that instead or even use the IP of the instance.

The advantage is that oracle free tiers is free forever, you get few resources and can create according, usually two x86 instances (single core multi threaded 2xvcpu)  1gb ram an up to 200GB of storage that you can split as you like between boot and storage volumes. 

It is not that trivial to do it, but is feasible, If anyone is interested, just ask and I can create a guideline.

Cheers.
